### PR TITLE
Bug 1189337 - Remove footer + Sign Out button on settings page

### PR DESCRIPTION
**Pull request by shahbaz17**
Bug fixed. Review this change

### CODE REVIEW

**Comment by spasovski** [comment]
The bug doesn't mention why we're doing this. This PR just removes the two signout/signin links.

**Comment by shahbaz17** [comment]
Hi spasovski i am working on this bug https://bugzilla.mozilla.org/show_bug.cgi?id=1189337

**Comment by shahbaz17** [comment]
Can you help me ? Why i'm getting these errors

**Comment by spasovski** [comment]
@shahbaz17 the errors can be ignored for now since they are related to a large refactor last week. That bug has no details apart from its title so I am confused by what the intention of it is.

**Comment by ngokevin** [comment]
problem we didn't merge this last time because it didn't address tests. tests are all failing now so we should get that fixed first

File: src/templates/settings.html [hunk 1]
```diff
@@ -42,8 +42,6 @@
       <div class="account-settings-save">
         <button class="button hide-logged-out" type="submit">{{ _('Save Changes') }}</button>
         <a href="#" class="button persona register hide-logged-in">{{ _('Register') }}</a>
-        <a class="button action login persona hide-logged-in">{{ _('Sign In') }}</a>
```
Comment (ngokevin) [inline src/templates/settings.html]: not supposed to remove this

**Comment by shahbaz17** [comment]
So what can i do now? any suggestion?

**Comment by shahbaz17** [comment]
hi ngokevin i have updated the code. Can u look into this

**Comment by shahbaz17** [comment]
@ngokevin Thanks for the detailed description about this bug. Looking forward to fix more bug

**Pull request merged**